src/framework/scene-grab.js

```diff
@@ -153,9 +153,9 @@ class SceneGrab {
                 if (camera.renderSceneColorMap) {
 
                     // allocate / resize existing RT as needed
-                    if (this.colorRenderTarget?.width !== device.width || this.colorRenderTarget?.height !== device.height) {
+                    if (this.colorRenderTarget?.width !== camera.renderTarget?.colorBuffer?.width || this.colorRenderTarget?.height !== camera.renderTarget?.colorBuffer?.height) {
```

Comment:
this won't work in case camera.renderTarget is null/undefined.
I'd suggest to create a compare function, which takes two render targets and a device, compares it all ... and use it 4 times in this file.

Reply:
test it with the grab-pass engine example and resize the window - that likely does not work